src/winforms/toga_winforms/widgets/textinput.py

```diff
@@ -1,21 +1,29 @@
-from travertino.size import at_least
-
 from toga_winforms.libs import WinForms
+from travertino.size import at_least
 
 from .base import Widget
 
 
+class TogaTextInput(WinForms.TextBox):
+    def __init__(self, interface):
+        super().__init__()
+        self.Multiline = False
+        self.interface = interface
+
+    def on_text_change(self, sender, event):
+        if self.interface._on_change:
+            self.interface.on_change(self.interface)
```

Comment:
Does this need to be a whole standalone class? Could it just be an on_text_change method on the *Toga* TextInput widget?

Reply:
No, it doesn't :) Changed to a single class.